### ISSUE TITLE
Upgrading Windows 10 Deleted All Data

### ISSUE THREAD

**Issue by abow** [dfahlander/Dexie.js#983]
Yesterday I upgraded Windows 10 to version 1909.  I opened up my app in Chrome for the first time since the upgrade and see that all my data has been deleted.  

Is this a common issue?  

Is there anyway to avoid it?

**Comment by swedenborg** [comment]
Yes . You should ``` export import ``` when changing browsers, the indexedDB that Dexie uses is local to the browser.

https://dexie.org/docs/ExportImport/dexie-export-import

https://codepen.io/dfahlander/pen/RqwoaB/

**Comment by abow** [comment]
I didn't change browsers, though.

**Comment by swedenborg** [comment]
ahhh . yes . the "Windows 10 version upgrade" would wipe your filesystem so the browser do not have the handle to the wiped instance. 

Different browsers on same OS ex. chrome, edge, opera, firefox etc ... are all locally separated to DB instance's. Separate browser = separate DB.

**Comment by abow** [comment]
I don't understand.  It didn't wipe my file system.  It was just a Windows 10 update.

**Comment by swedenborg** [comment]
oohhh . i thought clean install and can not answer on what update and migration of apps-preferences for W-10 do or not ... it is the chrome preferences and files that should be preserved on the update is what one expect ? . please check for more info on the specifics on W-10 related sites and forums.

**Comment by l27-0-0-1** [comment]
Might be the chrome update https://www.theregister.co.uk/2019/12/16/chrome_79_update_a_catastrophe_for_android_users_with_webview_apps/
